src/main/webapp/partials/instructorHelpSearch.jsp

```diff
@@ -0,0 +1,365 @@
+                                                </div>
+                                            </td>
+                                            <td><span class="highlight">Section</span> C</td>
+                                            <td><span class="highlight">Team</span> C</td>
+                                            <td>Jack Wayne</td>
+                                            <td class="align-center">Joined</td>
+                                            <td><span class="highlight">jack@email.com</span></td>
+                                            <td class="no-print align-center">
+                                                <a class="btn btn-default btn-xs" title="View details of the student"
+                                                    href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">View</a>
+                                                <a class="btn btn-default btn-xs" title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly"
+                                                   href="" target="_blank" rel="noopener noreferrer" data-toggle="tooltip" data-placement="top">Edit</a>
+                                                <a class="course-student-delete-link btn btn-default btn-xs"
+                                                title="Delete the student and the corresponding submissions from the course" href="" data-toggle="tooltip" data-placement="top">Delete</a>
+                                                <a class="btn btn-default btn-xs" href="" title="View all data about this student" target="_blank" rel="noopener noreferrer"
+                                                   data-toggle="tooltip" data-placement="top">All Records</a>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </li>
+        <li>
+            <span class="text-bold">
+                <a name="searchCommentForResponses">
+                    <h3>2. Searching for comments on responses</h3>
+                </a>
+            </span>
+            <div>
+                You can search for your comments on responses given by students. To do this check
+                <b>'Comments for responses'</b> below the search box and type your keywords. When done hit <b>'Search'</b>.
+                <br>Suppose you entered <b>'good'</b> in the search box and let's assume relevant data exists, then the search results would look something similar to this -
```

Comment:
`Suppose you searched for 'good'. Assuming the relevant data exists, the search results...`